src/Robo/Plugin/Commands/DrushCommands.php

```diff
@@ -11,7 +11,7 @@ class DrushCommands extends \BluesparkLabs\Spark\Robo\Tasks {
   private $commandBase;
 
   private function prepare() {
-    $this->isContainer = $this->containerExists('php');
```

Comment:
If we're removing this call, then is the containerExists method even needed anymore?  (I can't remember if it was used anywhere else or not).

Reply:
It's not used anywhere else, but I wanted to keep it anyway, because it may be useful in other scenarios and it doesn't hurt having it.